README.md

```diff
@@ -6,7 +6,6 @@
 [![Zenodo Badge](https://zenodo.org/badge/18918/sympy/sympy.svg)](https://zenodo.org/badge/latestdoi/18918/sympy/sympy)
 [![codecov Badge](https://codecov.io/gh/sympy/sympy/branch/master/graph/badge.svg)](https://codecov.io/gh/sympy/sympy)
 
-[![SymPy Banner](banner.svg)](https://sympy.org/)
```

Comment:
I think there is no need for removing this line here. You can skip this line while reading it in `setup.py`